dynawo/sources/Modeler/DataInterface/DYNCriteria.cpp

```diff
@@ -416,8 +410,6 @@ GeneratorCriteria::checkCriteria(double t, bool finalStep, const boost::shared_p
                                                                                                           params_->getPMin(),
                                                                                                           params_->getId()));
           distanceToGeneratorFailingCriteriaMap.insert({generatorFailingCriteria->getDistance(), generatorFailingCriteria});
-          Message mess = DYNLog(SourceUnderPower, (*it)->getID(), p, params_->getPMin(), params_->getId());
```

Comment:
This log is deleted because it will be put in failingCriteria_ with the printAllFailingCriteriaIntoLog function below (l.426) for LOCAL_VALUE criteria type and l.443 and l.461 for SUM criteria type.